sortingQuadraticComplexity/sortingQuadraticComplexity/main.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+
+namespace Sort
+{
+    class Program
+    {
+        public static void bubbleSort(int[] arrayOut)
```

Comment:
Не, в .NET методы всегда называются с заглавной:
```suggestion
        public static void BubbleSort(int[] arrayOut)
```